prometheus/promhttp/instrument_client.go

```diff
@@ -49,21 +49,23 @@ func InstrumentRoundTripperInFlight(gauge prometheus.Gauge, next http.RoundTripp
 // http.RoundTripper to observe the request result with the provided CounterVec.
 // The CounterVec must have zero, one, or two non-const non-curried labels. For
 // those, the only allowed label names are "code" and "method". The function
-// panics otherwise. Partitioning of the CounterVec happens by HTTP status code
+// panics otherwise. For the "method" label a predefined default label value set
+// is used to filter given values. `additionalMethods` can be used to add more
+// methods to the set. Partitioning of the CounterVec happens by HTTP status code
```

Comment:
```suggestion
// panics otherwise. For the "method" label a predefined default label value set
// is used to filter given values. Values outside of predefined values will count as `unknown` method. `WithExtraMethods` can be used to add more
// methods to the set. Partitioning of the CounterVec happens by HTTP status code
```